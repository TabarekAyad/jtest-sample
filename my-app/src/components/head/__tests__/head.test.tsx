import * as React from "react";
import * as renderer from "react-test-renderer";
import Head from '../head'

it("this is working head", () => {
  const tree = renderer.create(<Head/>).toJSON();
  expect(tree).toMatchSnapshot();
});
