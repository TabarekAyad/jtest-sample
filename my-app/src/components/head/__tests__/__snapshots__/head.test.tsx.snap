// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this is working head 1`] = `
<h1>
  adfs
</h1>
`;
